dev-docs/modules/haloRtdProvider.md

```diff
@@ -16,21 +16,24 @@ sidebarType : 1
 * TOC
 {:toc}
 
-Audigent is a next-generation data management platform and a first-of-a-kind
-"data agency" containing some of the most exclusive content-consuming audiences
-across desktop, mobile and social platforms.
-
-This real-time data module provides an integrated post-cookie Halo identity and
-real-time user segmentation solution that seamlessly integrates into your bid
-request cycle. Users are identified and bid request objects are augmented
-with first-party data derived segments prior to being sent to exchange/ssp
-services in order to optimize targeting and increase publisher revenue.
-
-Audigent maintains a large database of first-party Tradedesk Unified ID,
-Audigent Halo ID and other id provider mappings to various third-party segment
-types that are utilizable across different backends. With the Halo RTD module,
-these segments and other data can be retrieved and utilized by supporting
-exchange and SSP backends in real-time during your bid request cycle.
+Audigent is a next-generation, 1st party data management platform and the
+world’s first "data agency", powering the programmatic landscape and DTC
+eCommerce with actionable 1st party audience and contextual data from the
+world’s most influential retailers, lifestyle publishers, content creators,
```

Comment:
As a general rule tech docs should not contain any marketing style language, particularly for an open-source product like Prebid where we have to remain agnostic on products. Can you change 'data from the world's most influential retailer...'

to: 

date from retailers, lifestyle publishers...